src/map-interactions.react.js

```diff
@@ -325,7 +326,9 @@ export default class Interactions extends Component {
   render() {
     return (
       <div
-        ref="container"
+        ref={container => {
```

Comment:
Interesting...but I guess that would make sense. Should we do that here and in V3 as well, @Pessimistress?